API-reference.md

```diff
@@ -122,12 +122,15 @@ The `IDependencyTelemetry` interface is described below
 Parameter | Type | Description
 ---|---|---
 `id` | string | **Required**<br>Unique id, this is used by the backend to correlate server requests.
-`responseCode` | number | **Required**<br>Response code returned by the dependency request (e.g., `200` for a success)
-`absoluteUrl?` | string | **Optional**<br>Absolute url used to make the dependency request
-`success?` | boolean | **Optional**<br>Whether or not the request was successful or not (e.g., `responseCode` in the range 200-299)
-`commandName?` | string| **Optional**<br>Command used to make the dependency request
+`responseCode` | number | **Required**<br>Response code returned by the dependency request (e.g., `200` for a success).
+`name?` | string | **Optional**<br>Name of the command initiated with this dependency call.
 `duration?` | number | **Optional**<br>Elapsed time of request & reply
-`method?` | string | **Optional**<br>Represents request verb (GET, POST, etc.)
+`success?` | boolean | **Optional**<br>Whether or not the request was successful or not (e.g., `responseCode` in the range 200-299)
+`startTime?` | Date | **Optional**<br>Dependency start timestamp.
+`correlationContext?` | string | **Optional**<br>correlation context from the server.
+`type?` | string | **Optional**<br>Dependency type name.
+`data?` | string | **Optional**<br>Command initiated by this dependency call.
```

Comment:
While this is defined on the interface, it doesn't seem that it's supported as a user settable field. It's defined on the interface so the internal code has correct type checking.

So comment should either be removed or changed to **Internal**